0004-ecs2ext-protocol/README.md

```diff
@@ -218,39 +172,49 @@ The NATS message payload is an Avro object with the following schema ([file](./e
         "string",
         "null"
       ],
-      "default": "",
       "doc": "Unique endpoint ID. Null for endpoint-unaware extensions."
     },
     {
-      "name": "path",
+      "name": "resourcePath",
+      "type": "string",
+      "doc": "Action path field that is used to determine the message handling function and the payload format."
+    },
+    {
+      "name": "requestId",
       "type": [
-        "string",
+        "int",
         "null"
       ],
-      "doc": "Action path field that is used to determine the message handling function and the payload format. Null if extension responding with error status code with no need to publish message to client"
+      "doc": "Request ID used by client to match responses."
     },
     {
       "name": "payload",
       "type": [
         "bytes",
         "null"
       ],
-      "doc": "Serialized message content. Can be null in status-only messages."
+      "doc": "Serialized message content."
     },
     {
       "name": "statusCode",
+      "type": "int",
+      "default": 200,
+      "doc": "Message processing status code."
+    },
+    {
+      "name": "reasonPhrase",
       "type": [
-        "int",
-        "null"
+        "null",
+        "string"
       ],
-      "default": 200,
-      "doc": "Message processing status code. Can be null in requests."
+      "default": null,
+      "doc": "Is intended to give a short textual description of the statusCode."
```

Comment:
Discussed locally. It is used in MQTT, too, as defined in 1/KP error response format.